code/src/components/subintroElement/SubintroElement.tsx

```diff
@@ -0,0 +1,18 @@
+import Box from "@mui/material/Box";
+import AppTypography from "@/components/app-typography/AppTypography";
+
+type SubintroElement = {
+    content: string;
+    icon: string;
+}
+
+const SubintroElement = ({ content, icon }: SubintroElement) => {
+    return (
+        <Box className="spa-subintro_element" data-testid="spa-subintro-item">
```

Comment:
We should follow [BEM](https://getbem.com/) methodology for classes

Reply:
What is wrong with the naming of this class?

Reply:
We should `spa-subintro__element`

Reply:
Oh, its double underscore. I got you.

Reply:
Done